freqtrade/exchange/exchange.py

```diff
@@ -1290,6 +1290,10 @@ async def _async_get_historic_ohlcv(self, pair: str, timeframe: str,
                     p, _, new_data = res
                     if p == pair:
                         data.extend(new_data)
+        # Some exchanges show data past the current time and set all values to null.
+        current_time = arrow.utcnow().int_timestamp * 1000
+        data = filter(lambda item: item[0] < current_time, data)
```

Comment:
TBH - i'm not certain this is something that should be fixed at freqtrade level.

Please open this as issue or PR against ccxt. This would be the only exchange this has been observed - so i think it makes more sense to fix this at ccxt - so other projects won't run into the same problem.

I'm happy to merge this only if ccxt explicitly says they're refusing to have this in their code.

Reply:
I'll do that. I was debating whether or not this is a ccxt issue. I'll try it there's today!

Reply:
that's not always clear - i always try to think "will this be a problem for other projects using ccxt, too" - and if the answer is yes - i try to fix it there.

As you'll notice above - i'm not per se excluding this from being added - just think it'll make more sense in ccxt.

Reply:
@peterwilli any update on this?

Reply:
> @peterwilli any update on this?

Hey xmatthias I'm sorry I got distracted with another crypto coin. I'll resubmit a PR today to CCXT about the downloading part.